sklearn/datasets/_samples_generator.py

```diff
@@ -740,6 +741,9 @@ def make_blobs(n_samples=100, n_features=2, centers=None, cluster_std=1.0,
         for reproducible output across multiple function calls.
         See :term:`Glossary <random_state>`.
 
+    return_centers: bool, optional (default=False)
+        If True, then return the centers of each cluster
+
     Returns
```

Comment:
Please update this section too

Reply:
Why did you resolve this? I don't see any changes under Returns

Reply:
I thought you were also mentioning about the versionadded directive. I could have cross checked it. sorry.